eng/illink.targets

```diff
@@ -83,6 +83,8 @@
       <ILLinkArgs>$(ILLinkArgs) --skip-unresolved true</ILLinkArgs>
       <!-- keep interface implementations -->
       <ILLinkArgs>$(ILLinkArgs) --disable-opt unusedinterfaces</ILLinkArgs>
+      <!-- disable optimization to unblock dependencies bump -->
+      <ILLinkArgs>$(ILLinkArgs) --disable-opt ipconstprop</ILLinkArgs>
```

Comment:
maybe file an issue for reenabling this optimization?

Reply:
#31712 31712